examples/client/client.go

```diff
@@ -72,54 +73,60 @@ func sendPostRequest() {
 	req.Header.SetMethod(fasthttp.MethodPost)
 	req.Header.SetContentTypeBytes(headerContentTypeJson)
 	req.SetBodyRaw(reqEntityBytes)
+
 	resp := fasthttp.AcquireResponse()
 	err := client.DoTimeout(req, resp, reqTimeout)
 	fasthttp.ReleaseRequest(req)
-	if err == nil {
-		statusCode := resp.StatusCode()
-		respBody := resp.Body()
-		fmt.Printf("DEBUG Response: %s\n", respBody)
-		if statusCode == http.StatusOK {
-			respEntity := &Entity{}
-			err = json.Unmarshal(respBody, respEntity)
-			if err == io.EOF || err == nil {
-				fmt.Printf("DEBUG Parsed Response: %v\n", respEntity)
-			} else {
-				fmt.Fprintf(os.Stderr, "ERR failed to parse response: %v\n", err)
-			}
-		} else {
-			fmt.Fprintf(os.Stderr, "ERR invalid HTTP response code: %d\n", statusCode)
-		}
-	} else {
+	defer func() { fasthttp.ReleaseResponse(resp) }()
```

Comment:
```suggestion
	defer fasthttp.ReleaseResponse(resp)
```
You don't have to wrap this in a closure.

Reply:
Really, my mistake
Thanks for the comment :)